### ISSUE TITLE
Function GetPackageFamilyName not found in kernel33.dll when trying to run Guilded.exe

### ISSUE THREAD

**Issue by Arkanu-of-Galatiel** [vxiiduu/VxKex#91]
I attempted to run Guilded using VxKex (Guilded dropped windows 7 support, bricking my guilded install and the auto-updating prevents installing older versions), and I got this error log:
https://paste.ee/p/MHkN2

System: Windows 7 Home Premium 64-bit
CPU: AMD FX-8350

I put the CPU in just in case it happened to be relevant.

**Comment by CamrynSM36** [comment]
yeah this issue doesnt just effect Windows 7. it also effects Windows Vista/Windows Server 2008 with the Extended Kernel yes. it did previously work with the extended kernel. but i got the same error. and yes i did try w7 with VxKex too. Guilded just doesnt throw an error at all under windows 8.1 it just wont start.
